packages/portfolio-planner/src/engine.ts

```diff
@@ -31,26 +36,24 @@ const encodedKeyToPath = (key: string) => {
   return path;
 };
 
-export const startEngine = async ({
-  agd,
-  rpc,
-  spectrum,
-  cosmosRest,
-}: {
+type IO = {
   agd: CosmosCommand;
   rpc: CosmosRPCClient;
   spectrum: SpectrumClient;
   cosmosRest: CosmosRestClient;
-}) => {
+};
+
+export const startEngine = async ({ agd, rpc, spectrum, cosmosRest }: IO) => {
   await null;
-  let status = await (await agd.exec(['status'])).stdout;
+  let chainStatus = await (await agd.exec(['status'])).stdout;
```

Comment:
FYI I think we should consider getting the status from the RPC, and deprecate agd.